posts/test_comment.py

```diff
@@ -0,0 +1,119 @@
+import pytest
+from .models import Resume
+from .models import Comment
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+
+# Post and post creator details
+USERNAME = "testuser"
+FIRSTNAME = "Test"
+LASTNAME = "User"
+PASSWORD = "testpass"
+EMAIL = "testuser@gmail.com"
+DESCRIPTION = "this is a test resume"
+FILE1 = "Alon_Shakaroffs_resume.pdf"
+
+# Comment and comment creator details
+COMMENT_USERNAME = "testCommentUser"
+COMMENT_FIRSTNAME = "TestComment"
+COMMENT_LASTNAME = "UserComment"
+COMMENT_PASSWORD = "testpassComment"
+COMMENT_EMAIL = "testcomment@gmail.com"
+COMMENT = "Very nice ResuMe!"
+A_VERY_LONG_COMMENT = """
+                        test test test test test test test test
+                        test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test test test test
+                        test test test test test test test test test test test test test test test test test test test
+                    """
+
+# -----------------------------------------------fixtures----------------------------------------------------------
+
+
+@pytest.fixture
+def post_creator():
```

Comment:
The creator is the author?

Reply:
> The creator is the author?

yes I will change the name to post author for readability.

Reply:
Looks like there are still variables and functions with `creator` in their name